### DISCUSSION TITLE
Import issue- attachment broken [Oct 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4166] [Discord Threads]
> 🙋‍♂️ **cirsten**  
#### PS: the copy project info button doesn't seem to work from my chrome browser!  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1028057902243655772/Screenshot_20221008-072226_Chrome-01.jpg)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### What version of NocoDB are you on? & what is the attachment type in this particular case?  
#  
> 🙋‍♂️ **cirsten**  
#### V0.97.0. images: JPG and PNG  
#  
> 🙋‍♂️ **cirsten**  
#### Thanks for your help  
#  
> 🙋‍♂️ **cirsten**  
#### PostgreSQL  
#  
> 💁 **o1lab**  
#### and are you behind a reverse proxy or something ?  
#  
> 🙋‍♂️ **cirsten**  
#### No.   
#### It's installed via Kubernetes container (on a TrueNAS Scale system)  
#  
> 💁 **o1lab**  
#### Can u right click on the image and open it in a new browser tab ? if its missing https - add https and see if that help  
#  
> 🙋‍♂️ **cirsten**  
#### No matter what I try, right clicking does not give an option to open in a new browser window   
#### Also each image file has had a 6 letter combination added to the start of the file name. Is that part of the import process, or an error?  
#  
> 💁 **o1lab**  
#### you can see network traffic and see what is the URL of the image  
#  
> 🙋‍♂️ **cirsten**  
#### not sure how to do that?  
#  
> 🙋‍♂️ **cirsten**  
#### do you mean "view page source"?  
#  
> 🙋‍♂️ **cirsten**  
#### If I click on an image and view source, this is what it get:  
#  
> 🙋‍♂️ **cirsten**  
#### <!DOCTYPE html>   
#### <html >   
####    
#### <head >   
####   <link rel="stylesheet" href="./_nuxt/entry.2fb2c179.css">   
#### </head>   
####    
#### <body >   
####   <div id="__nuxt"></div><script>window.__NUXT__={serverRendered:false,config:{public:{},app:{baseURL:"/",buildAssetsDir:"/_nuxt/",cdnURL:"."}}}</script><script type="module" src="./_nuxt/entry-caaaf3ce.mjs"></script>   
#### </body>   
####    
#### </html>  
#  
> 💁 **o1lab**  
####   
![image0](https://cdn.discordapp.com/attachments/1028057902449184788/1028805140708339812/Screenshot_2022-10-10_at_00.03.45.png)  
#  
> 💁 **o1lab**  
#### if you familiar with looking up API requests - you can see what API request and response will be  
#  
> 🙋‍♂️ **cirsten**  
#### not familiar with API requests at all, sorry  
#  
> 🙋‍♂️ **cirsten**  
#### I could always give access to the DB? The only thing on there is this test DB.  
#  
> 💁 **o1lab**  
#### pls DM  
#  
> 💁 **o1lab**  
#### I will see later tomorrow  
#  
> 🙋‍♂️ **cirsten**  
#### Done  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1028057902449184788)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
